scrapy/core/scraper.py

```diff
@@ -333,6 +329,19 @@ def _process_spidermw_output(
             )
         return None
 
+    def start_itemproc(self, item, *, response: Optional[Response]):
```

Comment:
```suggestion
    def start_itemproc(self, item, *, response: Optional[Response]) -> Deferred[Any]:
```

Reply:
(this most likely needs reformatting)

Reply:
Did not need it in the end (by 2 chars).